apps/web/src/styles/7-components/InfoBox/InfoBox.scss

```diff
@@ -0,0 +1,14 @@
+@use "govuk/settings/_colours-applied" as colors;
+
+$pi-govuk-background-colour:  colors.govuk-colour("light-grey");
+$pi-color: rgb(11, 12, 12);
```

Comment:
Same, there is no need for a Sass var as we are not reusing this anywhere.

Reply:
👍